explainability-service/src/main/java/org/kie/trustyai/service/data/storage/Storage.java

```diff
@@ -1,11 +1,13 @@
 package org.kie.trustyai.service.data.storage;
 
-import java.nio.file.Path;
+import org.kie.trustyai.service.data.exceptions.StorageWriteException;
 
 public abstract class Storage implements StorageInterface {
+    public DataFormat getDataFormat() {
+        return DataFormat.CSV;
```

Comment:
Is there a reason why we default to `CSV`?

Reply:
nope, that's not needed, removed